pkg/tfgen/generate_nodejs.go

```diff
@@ -811,7 +811,12 @@ func (g *nodeJSGenerator) emitTypeScriptProjectFile(pack *pkg, files []string) e
         "module": "commonjs",
         "moduleResolution": "node",
         "declaration": true,
-        "sourceMap": true
+        "sourceMap": true,
+        "stripInternal": true,
+        "experimentalDecorators": true,
+        "noFallthroughCasesInSwitch": true,
+        "forceConsistentCasingInFileNames": true,
+        "strict": true
```

Comment:
`strict: true` enables the gamut of strict tests.  I want this so that i can have my mixin code strictly analyzed